CONTRIBUTING.md

```diff
@@ -4,6 +4,20 @@
 
 In this project, we are heavily utilising GitHub features to document and signal any progress in the website development.
 
+### Finding or creating issues
+
+Most contributions start from defining issues. Anyone can open an issue for discussion. You can head to [this link](https://guides.github.com/features/issues/) for deep understanding about Issues. Specifically, you can start finding several Issues in [here](https://github.com/kawalcovid19/wargabantuwarga.com/issues). There are only 2 categories in Issues section, Open and Closed.
+
+### Open Issues
+
+Open Issues means it still needs more attention and need to be resolved. Commonly, contributors will explore Open Issue section to resolve an issue.
+
+### Closed Issues
+
+Closed Issues means it has been completed or no need further action. Closed issue can be changed to open if contributors find related issue with the same category in the future.
+
+Please pay attention on every issue attribute. Every issue might be referenced by other contributors through [Linked Pull Request](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). It means the issue has been being handled. To avoid working on the same issue, contributors were strongly encouraged to submit [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) first.
```

Comment:
```suggestion
Please pay attention on every issue attribute. Every issue might be referenced by other contributors through [Linked Pull Request](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). It means the issue is currently being handled. To avoid working on the same issue, contributors are strongly encouraged to submit a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) when they start working on an issue.
```